Signal.xcodeproj/project.pbxproj

```diff
@@ -101,9 +100,24 @@
 		346129FF1FD5F31400532771 /* OWS103EnableVideoCalling.m in Sources */ = {isa = PBXBuildFile; fileRef = 346129F21FD5F31400532771 /* OWS103EnableVideoCalling.m */; };
 		34612A001FD5F31400532771 /* OWS105AttachmentFilePaths.h in Headers */ = {isa = PBXBuildFile; fileRef = 346129F31FD5F31400532771 /* OWS105AttachmentFilePaths.h */; };
 		34612A011FD5F31400532771 /* OWS104CreateRecipientIdentities.h in Headers */ = {isa = PBXBuildFile; fileRef = 346129F41FD5F31400532771 /* OWS104CreateRecipientIdentities.h */; };
+		34612A061FD7238600532771 /* OWSContactsSyncing.h in Headers */ = {isa = PBXBuildFile; fileRef = 34612A041FD7238500532771 /* OWSContactsSyncing.h */; };
+		34612A071FD7238600532771 /* OWSContactsSyncing.m in Sources */ = {isa = PBXBuildFile; fileRef = 34612A051FD7238500532771 /* OWSContactsSyncing.m */; };
 		346B66311F4E29B200E5122F /* CropScaleImageViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 346B66301F4E29B200E5122F /* CropScaleImageViewController.swift */; };
 		3471B1DA1EB7C63600F6AEC8 /* NewNonContactConversationViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 3471B1D91EB7C63600F6AEC8 /* NewNonContactConversationViewController.m */; };
 		3472229F1EB22FFE00E53955 /* AddToGroupViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 3472229E1EB22FFE00E53955 /* AddToGroupViewController.m */; };
+		347850311FD7494A007B8332 /* dripicons-v2.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 34330A5B1E787A9800DF2FB9 /* dripicons-v2.ttf */; };
+		347850321FD7494A007B8332 /* ElegantIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 34330A5D1E787BD800DF2FB9 /* ElegantIcons.ttf */; };
+		347850331FD7494A007B8332 /* fontawesome-webfont.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 34330A591E7875FB00DF2FB9 /* fontawesome-webfont.ttf */; };
+		3478504C1FD7496D007B8332 /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = B66DBF4919D5BBC8006EA940 /* Images.xcassets */; };
+		3478504D1FD74984007B8332 /* GIAG2.crt in Resources */ = {isa = PBXBuildFile; fileRef = 458D51421FCCD82400B5BC53 /* GIAG2.crt */; };
```

Comment:
Fonts, certs and xcassets are going in both apps.  I didn't bother with the old non-xcassets images.

Reply:
Are you copying the assets to both the framework bundle and the main app bundle? 

Or did you move them out of the main bundle to the framework bundle?

If it's the former, doesn't that mean we'll have duplicate assets - how much space does that cost us?

If it's the latter, don't we need to update all image-loading code to specify the framework bundle instead of the main bundle?

This overlaps with my work in https://github.com/WhisperSystems/Signal-iOS/pull/2851, FWIW I went the route of moving (not copying) the shared assets into the framework, and updating their usage to load them from the specific bundle, but I could be convinced to do it another way.

And I moved the certificates into the SSK bundle, since code in SSK requires them to exist.

Reply:
Great question.  Our fonts are ~350 KB and our xcassets are ~3.5 MB, so I'm not that worried about duplicating them.  But if it's easy not to duplicate them, I'm not opposed. I favor minimizing developer time, not bundle size.

Reply:
Due to app slicing, it'll actually be a bit smaller than that, so YOLO I guess.

I'll rebase #2851 to follow this approach

We can always revisit this later.